subsys/net/lib/coap/coap.c

```diff
@@ -441,9 +525,8 @@ static int decode_delta(uint8_t *data, uint16_t offset, uint16_t *pos, uint16_t
 	return ret;
 }
 
-static int parse_option(uint8_t *data, uint16_t offset, uint16_t *pos,
-			uint16_t max_len, uint16_t *opt_delta, uint16_t *opt_len,
-			struct coap_option *option)
+int parse_option(uint8_t *data, uint16_t offset, uint16_t *pos, uint16_t max_len,
```

Comment:
Why was the static removed?

Reply:
The `static` was removed from the implementation as the declaration was added at the top of the file (with `static`) but since the declaration was not needed this will be reverted.